stress/src/logs.rs

```diff
@@ -1,39 +1,10 @@
 use opentelemetry_appender_tracing::layer;
 use opentelemetry_sdk::logs::{LogProcessor, LoggerProvider};
 use tracing::error;
-use tracing_subscriber::{prelude::*, Layer};
+use tracing_subscriber::prelude::*;
 
 mod throughput;
 
-struct NoopEventVisitor;
```

Comment:
NoOpLogLayer and NoopEventVisitor are not used, so removing them,

Reply:
Do we not need them for the future?

Reply:
I don't see them used for stress test.  We already have the benchmark test for them here - https://github.com/open-telemetry/opentelemetry-rust/blob/04a800c1b6a7f096c5040864592edb2e0e8fb9db/opentelemetry-appender-tracing/benches/logs.rs#L83